src/main/java/org/spongepowered/common/config/SpongeConfig.java

```diff
@@ -473,6 +477,14 @@ public boolean chunkLoadOverride() {
         public void setChunkLoadOverride(boolean chunkLoadOverride) {
             this.chunkLoadOverride = chunkLoadOverride;
         }
+
+        public Locale getLocale() {
+            return Locale.forLanguageTag(this.locale);
```

Comment:
It's not restricted to Minecraft locales. All the LOCALE_CACHE does is cache the `String` to the `Locale`.